data.js

```diff
@@ -0,0 +1,36 @@
+var breakfastLocationData = [
+  {'name': 'Millionaire Club Charity Breakfast', 'hood': 'Belltown', 'address': '2515 Western Ave', 'meal': 'breakfast', 'days': [1, 2, 3, 4, 5], 'openTime': 6, 'closeTime': 7, 'restrictions': 'none'},
```

Comment:
it should be possible to flatten this data into one array of locations. there's a lot of duplicate information like name, address, hood for locations that have multiple meals. what if each location object had breakfast, lunch and dinner properties that were either `null` or `breakfast: { days: [1,2,3], openTime: 6, closeTime: 7, restrictions: 'none' }`.